doc/source/web.rst

```diff
@@ -1,7 +1,44 @@
 .. _web:
 
-Web
-===
+Streaming data from cloud sources
+=================================
+
+Spatial Temporal Asset Catalogs
+-------------------------------
+
+`STAC <https://stacspec.org/en>`_
+
+Stream Sentinel-2 level 2A data from Element84
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    from pathlib import Path
+    from geowombat.core.stac import open_stac
+    from rasterio.enums import Resampling
+
+    # Bounds in lat/lon WGS84
+    bounds = (left, bottom, right, top)
+
+    data, df = open_stac(
+        stac_catalog='element84',
```

Comment:
How do we find the stac_catalog names?

Reply:
I am not seeing 'element84' anywhere in the listed catalogs

Reply:
They are in classes under the `stac.py` module, which will be in the docs. I will add some lines that show
```python
from geowombat.core.stac import STACNames, STACCatalogs
print(list(STACNames))

[<STACNames.element84: 'element84'>,
 <STACNames.google: 'google'>,
 <STACNames.microsoft: 'microsoft'>]

print(STACNames.element84.value)
'element84'

print(getattr(STACCatalogs, STACNames.element84.value))
'https://earth-search.aws.element84.com/v0'
```

Reply:
Is this resolved? Auto added by sphinx?

Reply:
I do have `stac_catalog (str): Choices are ['element84', 'google', 'microsoft'].` in the `open_stac()` docstring. Is that what you mean?

I'll remove 'google' from that list.